README.md

```diff
@@ -17,24 +17,58 @@
     under the License.
 -->
 
-![Build Status](https://github.com/apache/trafficcontrol/workflows/CDN-in-a-Box%20CI/badge.svg)
-![Weasel License Checks](https://github.com/apache/trafficcontrol/workflows/Weasel%20License%20checks/badge.svg)
-![Traffic Ops Integration Tests](https://github.com/apache/trafficcontrol/workflows/Traffic%20Ops%20Go%20client/API%20integration%20tests/badge.svg)
-![Go Unit Tests](https://github.com/apache/trafficcontrol/workflows/Go%20Unit%20Tests/badge.svg)
-![Go Formatting](https://github.com/apache/trafficcontrol/workflows/Go%20Format/badge.svg)
-![TP Integration Tests](https://github.com/apache/trafficcontrol/workflows/TP%20Integration%20Tests/badge.svg)
-![TM Integration Tests](https://github.com/apache/trafficcontrol/workflows/TM%20Integration%20Tests/badge.svg)
+# Apache Traffic Control
+
+![](https://traffic-control-cdn.readthedocs.io/en/latest/_static/ATC-SVG-FULL-WHITE.svg)
```

Comment:
This is a lot of empty space. The website has [a version of the logo](https://trafficcontrol.apache.org/resources/Traffic-Control-Logo-FINAL-Black-Text.png) ([source](https://github.com/apache/trafficcontrol-website/blob/asf-site/resources/Traffic-Control-Logo-FINAL-Black-Text.png)) that includes the name of the project, too. Should that be used?

![Apache Traffic Control](https://trafficcontrol.apache.org/resources/Traffic-Control-Logo-FINAL-Black-Text.png)

Reply:
Ohh, oops the logo I used only works with the GitHub Dark theme. Will update this